main.go

```diff
@@ -98,6 +98,20 @@ func main() {
 			Name:  "no-cacerts",
 			Usage: "Skip CA certs population in settings when set to true",
 		},
+		cli.StringFlag{
+			Name:  "audit-log-path",
+			Usage: "Specifies the log path for Rancher Server API, not specifying means diasble server API audit log",
+		},
+		cli.IntFlag{
+			Name:  "audit-log-limit-size",
+			Value: 2048,
+			Usage: "Specified total size in megabytes all audit log files could use, audit log file will rotate each 100M.",
+		},
+		cli.IntFlag{
+			Name:  "audit-level",
+			Value: 0,
+			Usage: "Specified the audit level, 0 is metadata, level 1 include request body, level 2 include request body and response body",
```

Comment:
lets rephrase it a little: `Audit log level; 0 - log event metadata, 1 - log event metadata and request body, 2 - log event metadata, request body and response body`